src/style.css

```
@@ -0,0 +1,37 @@
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
```

you can combine selectors with coma
```suggestion
.stars--0 .stars__star:nth-child(-n + 0),
.stars--1 .stars__star:nth-child(-n + 1)  {
  background-image: url(images/star-active.svg);
}
```

---

fix all such cases